basicswap/basicswap.py

```diff
@@ -8770,7 +8770,7 @@ def redeemXmrBidCoinBLockTx(self, bid_id: bytes, cursor) -> None:
             lock_tx_depth = (chain_height - bid.xmr_b_lock_tx.chain_height) + 1
```

Comment:
```suggestion
            lock_tx_depth = (chain_height - bid.xmr_b_lock_tx.chain_height)
```

Confirmations are overstated by 1